tfjs-layers/src/layers/preprocessing/category_encoding_test.ts

```diff
@@ -0,0 +1,141 @@
+import { describeMathCPUAndGPU, expectTensorsClose} from '../../utils/test_utils';
+import { Tensor, tensor} from '@tensorflow/tfjs-core';
+import { CategoryEncoding } from './category_encoding';
+import * as utils from './preprocessing_utils';
+
+describeMathCPUAndGPU('Layer Output', () => {
```

Comment:
Include the name of the layer in the `describe`, e.g.

```suggestion
describeMathCPUAndGPU('Category Encoding', () => {
```